src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prev = initialValue;
+    let startIndex = 0;
+
+    if (arguments.length < 2) {
```

Comment:
Good work ! 
Just a suggestion I got from Paul 
that , Try to do such assignments using ternary operators.

   const prev = (arguments.length < 2)  ?  this[0]  :  initialValue;
   const startIndex = (arguments.length < 2)  ?  1  :  0;

Reply:
Okay, will do.

Reply:
Not done :)

Reply:
I committed it yesterday as well as the other task but it doesn't show up.